pulsar-common/src/main/java/org/apache/pulsar/client/impl/schema/SchemaInfoImpl.java

```diff
@@ -117,6 +117,6 @@ public SchemaHash getSchemaHash() {
 
     @Override
     public String toString() {
-        return SchemaUtils.jsonifySchemaInfo(this);
+        return SchemaUtils.jsonifySchemaInfo(this, false);
```

Comment:
is it necessary to change the behavior in this PR? It seems that previously the output was pretty printed, but this changes it to be without pretty printing.